internal/cli/cmd/login/login.go

```diff
@@ -47,12 +48,16 @@ type loginOptions struct {
 	ServerAddress string
 }
 
-func NewCommand() *cobra.Command {
-	cmdOpts := &loginOptions{}
+func NewCommand(
+	cfg libConfig.CLIConfig,
+) *cobra.Command {
+	cmdOpts := &loginOptions{
+		Config: cfg,
+	}
 
 	cmd := &cobra.Command{
-		Use:   "login SERVER_ADDRESS (--admin | --kubeconfig | --sso)",
-		Args:  option.ExactArgs(1),
+		Use:   "login [SERVER_ADDRESS] (--admin | --kubeconfig | --sso)",
```

Comment:
Would it better to show config address like below?

```suggestion
		Use:   "login [SERVER_ADDRESS | <default address here>] (--admin | --kubeconfig | --sso)",
```

Reply:
The current is fine, as the default address is not _actually_ being used as an argument but as a fallback.

Instead, it may be better to instruct people how to make use of the default address in the `Example` section.

Reply:
Added an example for omitted server address: https://github.com/akuity/kargo/pull/1975/commits/c327f72bc0988e058438070c06db921640f197a4